src/components/studies/scheduler/SchedulableSingleSessionContainer.tsx

```diff
@@ -92,6 +106,21 @@ const SchedulableSingleSessionContainer: FunctionComponent<SchedulableSingleSess
     setWindowErrors(windowErrorsArray)
   }, [sessionErrorState?.sessionWindowErrors])
 
+  React.useEffect(() => {
+    if (!sessionErrorState || sessionErrorState.notificationErrors.size == 0) {
+      setNotificationErrors([])
+      return
+    }
+    const notificationErrorsArray: notificationErrorArrayType[] = []
```

Comment:
while what you are doing it correct, I think map method is more appropriate because you are transforming one array to the other (sessionErrorState.notificationErrors -> notificationErrorsArray) so I think something like this would be clearer: 
```
const notificationErrorsArray =  sessionErrorState.notificationErrors.map((el, key) => ({  notificationName: 'notification ' + key,  notficationError: el }) )
 ```